.github/workflows/test.yml

```diff
@@ -30,9 +30,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - compose_version: '1.28.0'
+          - compose_version: '1.29.2'
             compose_path: '/usr/local/bin'
-          - compose_version: 'v2.0.1'
+          - compose_version: 'v2.2.3'
```

Comment:
I was going to suggest this, but now especially with https://github.com/getsentry/self-hosted/pull/1251#issuecomment-1015357276 ... what do you think of testing against more than two versions of Docker Compose? How about min/max of v1 and min/max of v2? So ...

- 1.28.0
- 1.29.2
- 2.0.1
- 2.2.3